content/en/tracing/version_tracking/_index.md

```diff
@@ -14,6 +14,75 @@ further_reading:
+- Error rate as a percentage of total requests.
+
+Additionally, fields can be added or removed from this overview table and your selections will be saved.  The additional fields are:
+
+- Endpoints that appear in a version that did not appear in the previous version.
+- Time active, showing the length of time from the first trace to the last trace sent to Datadog for that version.
+- Total number of Requests.
+- Total number of Errors.
+- Latency measured by p50, p75, p90, p95, p99, or the true max.
+
+{{< img src="tracing/version_tracking/VersionComparison.png" alt="Versions on the Service Page"  style="width:100%;">}}
+
+**Note:** The `version` section of of the service page will only appear if there is more than one version reporting in the selected time interval at the top of the page.
+
+## Version Comparison
+
+Clicking on any version row in the Version Summary table will open a detailed version comparison page, allowing you to compare two specific version of the same service.  By default, the comparison will compare the version clicked to the immediately previous version, but this can be changed to compare any two versions within the past 30 days.
+
+There are three main components of version comparison:
+
+- [Comparison Graphs](#comparison-graphs) for a visualization of requests and errors to services, useful for watching various types of [deployments](#version-deployment-strategies).
+- [Error Comparison](#error-comparison) to see errors that may have been introduced or solved by a deployment.
+- [Endpoint Comparison](#endpoint-comparison) to see how endpoint latency and error rates are performing in different versions of the service.
+
+### Comparison Graphs
+
+Similar graphs to the main service page will appear, however all other versions than the two being specifically compared will be combined and labelled as 'Other versions', allowing a quick overview of a deployment or difference in error rate to be quickly noticed.
+
+{{< img src="tracing/version_tracking/ComparisonGraphs.png" alt="Version Comparison Graphs" style="width:100%;">}}
+
+### Error Comparison
+
+This section will list error types that appear in either or both versions of the service.  While not a guarantee that errors have been introduced or solved, due to potentially limited traffic or other conditions for the error to occur, you can pivot from this page immediately into live or historical traces corresponding to this error for further investigation.
+
+{{< img src="tracing/version_tracking/ErrorComparison.gif" alt="Error Comparison"  style="width:100%;">}}
+
+### Endpoint Comparison
+
+This section shows all the endpoints for the service, along with a comparison between the performance of requests to these endpoints between the two versions.  The table can be sorted up or down according to latency, request volume or error rate to surface the most meaning potential performance improvement or degredation, and also filtered to show common or new endpoints.
```

Comment:
Seems a bit overcomplex, why not start with: "This section lets you compare the performance (requests, latency and errors) of each endpoint in the service. The table can be sorted by absolute values or by the change between versions allowing to easily detect large swings in latency or error rates or to validate that the highest throughput endpoints are still healthy following a deploy."